Persephone/Components/Browser/Album Browser/AlbumViewItem.swift

```diff
@@ -57,16 +59,18 @@ class AlbumViewItem: NSCollectionViewItem {
     }
   }
 
-  func setAppearance() {
+  func setAppearance(selected isSelected: Bool) {
     if #available(OSX 10.14, *) {
       let darkMode = NSApp.effectiveAppearance.bestMatch(from:
         [.darkAqua, .aqua]) == .darkAqua
 
       albumCoverView.layer?.borderColor = darkMode ? .albumBorderColorDark : .albumBorderColorLight
-      albumCoverBox.layer?.borderColor = NSColor.controlAccentColor.cgColor
+      albumCoverBox.layer?.borderColor = isSelected ? NSColor.controlAccentColor.cgColor : CGColor.clear
```

Comment:
Since all this depends on optional layers being present, could you add a `guard let` at the top of the method and then be able to just say `viewLayer.###` and `boxLayer.###` without unwrapping on every line?